CONTRIBUTORS.txt

```diff
@@ -87,18 +91,21 @@ needs to be listed here.
 - Norman Maurer <norman_maurer@apple.com>
 - Ole Begemann <ole@oleb.net>
 - OleG <emptystamp@gmail.com>
+- Peter Adams <pp_adams@apple.com>
 - Petro Rovenskyy <petro.rovenskyy@gmail.com>
 - Portia <portia-s@users.noreply.github.com>
 - Pushkar N Kulkarni <pushkar.nk@in.ibm.com>
 - Pushparaj Zala <pr92zala@gmail.com>
 - Richard Low <ral@apple.com>
 - Romain Pouclet <rpouclet@apple.com>
 - Rukshani Athapathu <arukshani@users.noreply.github.com>
+- Saleem Abdulrasool <compnerd@compnerd.org>
 - Sebastian Thiebaud <770074+SebastianThiebaud@users.noreply.github.com>
 - Semen Zhydenko <simeon.zhidenko@gmail.com>
 - Sergej Jaskiewicz <jaskiewiczs@icloud.com>
 - Sergo Beruashvili <beruashvili@gmail.com>
 - Shaun Hubbard <shaunhubbard2013@icloud.com>
+- Shekhar Rajak <60732970+shekhar-rajak@users.noreply.github.com>
```

Comment:
@shekhar-rajak if you tell me your real email address, I'm happy to make a `.mailmap` entry for it to show a real email address.

Reply:
@weissi , `shekhar_rajak@apple.com`. Thanks!

Reply:
thanks, done